ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -58,7 +58,7 @@ const styles = StyleSheet.create({
   }
 });
 
-export class TransactionDetailsScreen extends React.Component<Props> {
+class TransactionDetailsScreen extends React.Component<Props> {
```

Comment:
Not needed now but in general connected components export also the not-connected class so you can test the component without moking the store Provider passing the reduxinjected props directly to the not-connected component.

Reply:
I would say with export the component when actually needed :)